onos-classic/templates/statefulset.yaml

```diff
@@ -71,6 +71,8 @@ spec:
         env:
           - name: JAVA_OPTS
             value: -Xmx{{ .Values.heap }}
+          - name: ONOS_APPS
+            value: "org.opencord.sadis,org.opencord.olt,org.opencord.kafka"
```

Comment:
this shoudl probably be empty and allow overwrite. in the `values` file

Reply:
Yes this is what I did in the original version of this Helm chart IIRC